back/Dockerfile

```
@@ -1,5 +1,5 @@
 # protobuf build
-FROM --platform=$BUILDPLATFORM node:20.8-bullseye-slim AS proto-builder
+FROM --platform=$BUILDPLATFORM node:22.8-bullseye-slim AS proto-builder
```

```suggestion
FROM --platform=$BUILDPLATFORM node:22.16-bullseye-slim AS proto-builder
```

Let's use the latest version of the 22 branch!